src/client/app/components/GraphicRateMenuComponent.tsx

```diff
@@ -22,13 +23,10 @@ export default function GraphicRateMenuComponent() {
 	const dispatch = useDispatch();
```

Comment:
Most places use useAppDispatch but a few use the old useDispatch. Wondering why?

Reply:
The behavior 'should' be identical & not cause issues, however they all really should be using 'useAppDispatch'.  Must've been overlooked during translation.

Reply:
I replaced all uses with useAppDispatch.